src/components/progress-tracker/progress-tracker.pw.tsx

```diff
@@ -243,6 +246,23 @@ test.describe("Check props for ProgressTracker component", () => {
     );
   });
 
+  test("should position current value label to the left of tracker line when labelsPosition prop is left", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<ProgressTrackerComponent labelsPosition="left" />);
+
+    const currentValueLabel = progressTrackerMinVal(page);
```

Comment:
Question: Is there any reason why these `const` can't be pulled out to the top of the file as they're used in three test cases.